WeightProducer/python/MCSampleValues.py

```diff
@@ -999,141 +817,164 @@ class MCSampleValuesHelper():
             ),
             "BranchingRatio" : BRValues(
                 BR_13TeV=(1-(1-3*0.105)**2), BRSource_13TeV="http://pdg.lbl.gov/2019/reviews/rpp2018-rev-top-quark.pdf",
             ),
         },
         "ST_tW_top_5f_inclusiveDecays" : {
             "CrossSection" : XSValues(
-                XS_2016=38.09, XSSource_2016="XSDB (NLO)",
-                XS_2017=34.91, XSSource_2017="XSDB (NLO)",
-                XS_2018=34.91, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=32.45, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
         "ST_tW_antitop_5f_inclusiveDecays" : {
             "CrossSection" : XSValues(
-                XS_2016=38.06, XSSource_2016="XSDB (NLO)",
-                XS_2017=34.97, XSSource_2017="XSDB (NLO)",
-                XS_2018=34.97, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=32.51, XSSource_13TeV="GenXSecAnalyzer",
+            ),
+        },
+        "tZq_ll_4f_ckm" : {
+            "CrossSection" : XSValues(
+                XS_13TeV=0.07561, XSSource_13TeV="XSDB (unknown)",
             ),
         },
+        #TODO
         "tZq_W_lept_Z_hadron_4f_ckm" : {
             "CrossSection" : XSValues(
                 XS_2016=0.1573, XSSource_13TeV="XSDB (unknown)",
             ),
         },
+        #TODO
         "tZq_Zhad_Wlept_4f_ckm" : {
             "CrossSection" : XSValues(
                 XS_2017=0.1518, XSSource_2017="XSDB (unknown)",
             ),
         },
         "WW" : {
             "CrossSection" : XSValues(
-                XS_13TeV=51.723, XSSource_13TeV="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#Diboson, WW>4q NNLO",
+                XS_13TeV=75.95, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "WZ" : {
             "CrossSection" : XSValues(
-                XS_13TeV=47.13, XSSource_13TeV="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#Diboson, inclusive NLO from MCFM",
+                XS_13TeV=27.59, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "ZZ" : {
             "CrossSection" : XSValues(
-                XS_13TeV=16.523, XSSource_13TeV="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#Diboson, inclusive NLO from MCFM",
+                XS_13TeV=12.17, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "WWTo2L2Nu" : {
             "CrossSection" : XSValues(
-                XS_13TeV=12.178, XSSource_13TeV="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#Diboson, WW>2l2v NNLO",
+                XS_13TeV=11.09, XSSource_13TeV="XSDB (NLO)",
             ),
         },
         "WGJets_MonoPhoton_PtG-40to130" : {
             "CrossSection" : XSValues(
-                XS_2016=12.68,  XSSource_2016="XSDB (LO)",
-                XS_2017=12.93, XSSource_2017="XSDB (LO)",
-                XS_2018=12.95, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=19.77,  XSSource_13TeV="GenXSecAnalyzer",
```

Comment:
Any idea why this one is so much higher than before?

Reply:
Hmm, I don't see anything obvious. I reran the GenXSecAnalyzer and reobtained the same result.

Reply:
Take a look at the gen fragment for either the 2017 or 2018 sample and then compare it to the gen fragment for the new UL sample. Is it only the tune that changed? If that's the case, then you should ask the GEN POG if this change might be expected for this process.

Reply:
Yeah, sounds good.